arm-servicebus/2015-08-01/swagger/servicebus.json

```diff
@@ -1586,12 +1699,12 @@
       "x-ms-azure-resource": true,
       "description": "The Resource definition for other than namespace."
     },
-    "NamespaceListResult": {
+    "SBNamespaceListResult": {
```

Comment:
If you're planning to publish a new version of the sdk this should be okay. @olydis can you confirm this